turbo/generators/config.ts

```diff
@@ -0,0 +1,68 @@
+import { PlopTypes } from '@turbo/gen';
+
+export default function generator(plop: PlopTypes.NodePlopAPI): void {
+  plop.setGenerator('packages', {
+    description: 'Generate a base config for new package',
+    prompts: [
+      {
+        type: 'input',
+        name: 'name',
+        message:
+        'What is the name of the package?',
+        validate: (input) => {
+          if (!input.match(/^[a-z0-9]+(-[a-z0-9]+)*$/)) {
+            return 'Name must be in kebab-case and in lower case letters. Example: my-package';
+          }
+          return true;
+        },
+      },
+      {
+        type: 'input',
+        name: 'description',
+        message: 'Write a description for the package',
+      },
+    ],
```

Comment:
More inputs can be taken based on the new naming convention. Like choice of the `library`.